Makefile

```
@@ -495,6 +503,8 @@ release: clean-release $(RELEASE_DIR)  ## Builds and push container images using
 .PHONY: build-chart
 build-chart: $(HELM) $(KUSTOMIZE) $(RELEASE_DIR) $(CHART_RELEASE_DIR) $(CHART_PACKAGE_DIR) ## Builds the chart to publish with a release
 	$(KUSTOMIZE) build ./config/chart > $(CHART_DIR)/templates/rancher-turtles-components.yaml
+	$(KUSTOMIZE) build ./config/exp > $(CHART_DIR)/templates/rancher-turtles-exp-components.yaml
```

I think it makes sense to move kustomization/kustomizeconfig yAML files to new crd folder as well, since these are crd specific
```suggestion
	$(KUSTOMIZE) build ./config/exp/crd > $(CHART_DIR)/templates/rancher-turtles-exp-components.yaml
```

---

Current command includes everything in `./config/exp/*`, at the moment there are only CRDs but when we add controllers we're going to see RBAC also present there.